patches/redux-persist+6.0.0.patch

```diff
@@ -0,0 +1,14 @@
+diff --git a/node_modules/redux-persist/types/types.d.ts b/node_modules/redux-persist/types/types.d.ts
+index b3733bc..882f4de 100644
+--- a/node_modules/redux-persist/types/types.d.ts
++++ b/node_modules/redux-persist/types/types.d.ts
+@@ -42,7 +42,8 @@ declare module "redux-persist/es/types" {
+      */
+     getStoredState?: (config: PersistConfig<S, RS, HSS, ESS>) => Promise<PersistedState>;
+     debug?: boolean;
+-    serialize?: boolean;
++    serialize?: boolean | Function;
```

Comment:
this is the actual implementation but it was missing in the types definition and we needed to pass a function instead of a boolean to have our own serializer